model/events/synchronization.go

```diff
@@ -1,16 +1,16 @@
 package events
 
 import (
-	"github.com/onflow/flow-go/model/cluster"
 	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/model/messages"
 )
 
 type SyncedBlock struct {
 	OriginID flow.Identifier
-	Block    *flow.Block
+	Block    messages.UntrustedBlock
```

Comment:
Why are we using a value here? Have we considered using a pointer as before?

Reply:
Because the structure is pretty small for copying, and we don't need reference semantics. I don't have a strong opinion though - do you think a pointer is better here?